packages/core/src/PartialExecutionUtils/findStartNodes.ts

```diff
@@ -135,14 +135,14 @@ export function findStartNodes(options: {
 	graph: DirectedGraph;
 	trigger: INode;
 	destination: INode;
-	runData?: IRunData;
-	pinData?: IPinData;
+	pinData: IPinData;
+	runData: IRunData;
```

Comment:
would it make sense to define default values here, to not have to repeat these args in the tests?
```suggestion
	pinData: IPinData = {};
	runData: IRunData = {};
```

Reply:
I made them required so that we have to pass them in.

I actually forgot to do it:
![image](https://github.com/user-attachments/assets/1525ca0a-e8d4-4706-a28c-31c2bb598884)

This change makes it more explicit and less likely to forget it again.